### PR TITLE
Visual Sep, more Heli Ops, ML Coord requirements

### PR DESCRIPTION
**Pull request by mattkelly4**

### CODE REVIEW

**Review by alexd-332 — APPROVED** [review]

File: docs/terminal/sydney.md [hunk 1]
```diff
@@ -135,21 +135,79 @@ These can be displayed on vatSys using the `SY_VFR` map.
...
+Due to the low level of CTA (`A015`) in the BK CTR when **BK TWR** is offline, it is best practice to give airways clearance to aircraft at the holding point, to ensure departing aircraft can have uninterrupted climb.
 
 !!! example
-    **ABC** -> **SY TCU**: "Sydney Approach, ABC, PC12, POB 8, IFR, Taxiing YSBK for YSHL, Runway 11C" `AIP GEN 3.4`  
-    **SY TCU** -> **ABC**: "ABC, Sydney Approach, Squawk 3601, No Reported IFR Traffic, Call me Ready at the Holding Point for Airways Clearance"  
-    **ABC** -> **SY TCU**: "Squawk 3601, Wilco, ABC"  
-    **ABC** -> **SY TCU**: "ABC, Ready Runway 11C, Request clearance"  
-    **SY TCU** -> **ABC**: "ABC, Cleared to YSHL via ANKUB, Flight Planned Route. Make Visual right turn DCT ANKUB, Climb to A030"  
-    **ABC** -> **SY TCU**: "Cleared to YSHL via ANKUB, Flight Planned Route. Make Visual right turn DCT ANKUB, Climb to A030, ABC" 
+    **LOA**: "Sydney Approach, LOA, King Air IFR, taxiing YSBK for YSHL, runway 11C, 10 POB" `AIP GEN 3.4`
```
Comment (tylerthetiletiler) [inline docs/terminal/sydney.md:204]: "LOA, King Air, POB 10, IFR, Taxing Bankstown for Shellharbour, Runway 11C"

Correct praseology

File: docs/terminal/sydney.md [hunk 2]
```diff
@@ -135,21 +135,79 @@ These can be displayed on vatSys using the `SY_VFR` map.
...
-    **ABC** -> **SY TCU**: "Squawk 3601, Wilco, ABC"  
-    **ABC** -> **SY TCU**: "ABC, Ready Runway 11C, Request clearance"  
-    **SY TCU** -> **ABC**: "ABC, Cleared to YSHL via ANKUB, Flight Planned Route. Make Visual right turn DCT ANKUB, Climb to A030"  
-    **ABC** -> **SY TCU**: "Cleared to YSHL via ANKUB, Flight Planned Route. Make Visual right turn DCT ANKUB, Climb to A030, ABC" 
+    **LOA**: "Sydney Approach, LOA, King Air IFR, taxiing YSBK for YSHL, runway 11C, 10 POB" `AIP GEN 3.4`  
+    **SY TCU**: "LOA, Sydney Approach, squawk 3601, no reported IFR traffic, report ready at the holding point for airways clearance"  
+    **LOA**: "Squawk 3601, wilco, LOA"  
+
+    **ABC**: "LOA, ready Runway 11C"  
+    **SY TCU**: "LOA, cleared to YSHL via ANKUB, flight planned route, make visual right turn DCT ANKUB, climb to A030"
```
Comment (tylerthetiletiler) [inline docs/terminal/sydney.md:209]: Very rarely do they give visual departures with tower offline. 

"LOA, cleared to YSHL via ANKUB, flight planned route, Bankstown 8 Departure, climb via SID A030"

File: docs/terminal/sydney.md (hunk 1 above)
Comment (mattkelly4) [reply docs/terminal/sydney.md:204]: Fixed

File: docs/terminal/sydney.md (hunk 2 above)
Comment (mattkelly4) [reply docs/terminal/sydney.md:209]: Fixed

**Review by alexd-332 — APPROVED** [review]

**Review by chrisg06 — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Pull request merged**